modules/home-environment.nix

```diff
@@ -592,22 +591,36 @@ in
         ''
       else
         ''
+          function nixReplaceProfile() {
+            local oldNix="$(command -v nix)"
+
+            nix profile list \
+              | { grep 'home-manager-path$' || test $? = 1; } \
```

Comment:
I am a bit confused here: `'home-manager-path$'` is a regex, but `grep` is called without `-e`, how can this work?

Reply:
> The grep command offers three regex syntax options:
1. Basic Regular Expression (BRE)
2. Extended Regular Expressions (ERE)
3. Pearl Compatible Regular Expressions (PCRE)

> By default, grep uses the BRE syntax.

I had to look this up, as well.
https://phoenixnap.com/kb/grep-regex

Reply:
Wow, didn't know that, thank you!